lib/active_record/connection_adapters/mysql2rgeo_adapter.rb

```diff
@@ -66,7 +66,7 @@ class Mysql2RgeoAdapter < Mysql2Adapter
       # http://postgis.17.x6.nabble.com/Default-SRID-td5001115.html
       DEFAULT_SRID = 0
 
-      def initialize(connection, logger, connection_options, config)
+      def initialize(...)
```

Comment:
Required by this change to ActiveRecord: https://github.com/rails/rails/pull/44591/files#diff-538a9bf3450c3aed556a0d2eacc26839a60becd744cda8c0a247dcb06aa0f02fR44